qemu/tests/block_with_share_rw.py

```diff
@@ -22,7 +24,33 @@ def run(test, params, env):
     :param env: Dictionary with test environment.
     """
     msgs = ['"write" lock', 'Is another process using the image']
-    vm1 = env.get_vm(params["main_vm"])
+    modprobe_cmd = params.get('modprobe_cmd')
+    check_image_cmd = params.get('check_image_cmd')
+
+    # Some arm servers only have nvme controller
+    if params.get("start_vm") == "no":
+        if "arm64" in params["machine_type"]:
+            try:
+                process.run(check_image_cmd, shell=True)
+            except process.CmdError:
+                test.cancel("SCSI controller not available.")
+            else:
```

Comment:
suggest change to due to they look share same logic:

 if "arm64" in params["machine_type"]:
            try:
                process.run(check_image_cmd, shell=True)
            except process.CmdError:
                test.cancel("SCSI controller not available.")

  process.run(modprobe_cmd, ignore_status=True, shell=True)
  params["image_raw_device_stg"] = "yes"
  params["indirect_image_select_stg"] = "-1"
  params["start_vm"] = "yes"
  env_process.preprocess_vm(test, params, env,
                            params["main_vm"])
  vm1 = env.get_vm(params["main_vm"])

Reply:
good point, thanks